### PR TITLE
Changed view display

### PR DESCRIPTION
**Pull request by martinyde**
https://leantime.itkdev.dk/#/tickets/showTicket/3025
https://leantime.itkdev.dk/#/tickets/showTicket/3026
https://leantime.itkdev.dk/#/tickets/showTicket/3032

Skærmbillede 2024-11-21 kl  14 12 39

- Theme pager
- Change filter display
- Add filter for projects where user is contributer.
- Change table header

### CODE REVIEW

File: web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table--front.html.twig [hunk 1]
```diff
@@ -40,10 +40,11 @@
     sticky ? 'sticky-header',
     'table-auto',
     'w-full border-separate border-slate-400',
+    'min-w-[1000px]',
```
Comment (yepzdk) [inline web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table--front.html.twig:43]: Will this work on mobile?

File: web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig [hunk 2]
```diff
@@ -40,10 +40,10 @@
     sticky ? 'sticky-header',
     'table-auto',
     'w-full border-separate border-slate-400',
+    'min-w-[1000px]',
```
Comment (yepzdk) [inline web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig:43]: Will this work on mobile?

File: web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css [hunk 3]
```diff
@@ -75,13 +75,29 @@
 
 /* This css class is added on the projects view (admin/structure/views/view/projects) */
 .projects-view {
-  form {
-    display: flex;
+  .exposed-filter {
+    #edit-actions {
+      grid-column: span 1 / span 1;
+      margin-bottom: 1.25rem;
```
Comment (yepzdk) [inline web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css:81]: Can we reuse some spacing attributes from tailwind? Or create som custom properties resembling the tailwind equivalents, and the use them, so we ensure consistency.

File: web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig (hunk 2 above)
Comment (martinyde) [reply web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig:43]: The purpose is mobile with overflow-x-auto class. To force a scrollbar when the table becomes too small

File: web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table--front.html.twig (hunk 1 above)
Comment (martinyde) [reply web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table--front.html.twig:43]: The purpose is mobile

File: web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css (hunk 3 above)
Comment (martinyde) [reply web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css:81]: I don't have immediate access to the templates for these particular buttons, so we have to use css nesting. I'm not sure if it's possible to extend or somehow reuse the tailwind classes

File: web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig (hunk 2 above)
Comment (yepzdk) [reply web/themes/custom/itkdev/itkdev_project_theme/templates/views/views-view-table.html.twig:43]: Ahh - I see. Good work.

**Review by yepzdk — APPROVED** [review]

**Pull request merged**